File: observer/simple2ndorder_obs.py
# observer/simple2ndorder_obs.py
import control as co
import numpy as np
A = np.array([[0, 1],[-4, -0.2]])
C = np.array([[1], [0]]).T
L = co.place(A.T,C.T,[-2, -3]).T
print(L)

File: observer/simple2ndorder_obs.m
% observer/simple2ndorder_obs.m
% plant model
A = [0 1;-4 -0.2];
B = [0 1]';
C = [1 0];
sys = ss(A,B,C,0);

eig(A)

L = place(A',C',[-2,-3])'

eig(A-L*C)
